source/integrations/cloud-outgoing-webhooks.rst

```diff
@@ -138,14 +138,14 @@ Mattermost automatically translates the data coming from Slack:
    - ``<userid>``  to trigger a mention to a user, such as ``{"text": "<5fb5f7iw8tfrfcwssd1xmx3j7y> this is a notification."}``
    - ``<!channel>``, ``<!here>`` or ``<!all>`` to trigger a mention to a channel, such as ``{"text": "<!channel> this is a notification."}``
 
-2. The HTTP POST request body sent to a web service is formatted the same as Slack's. This means your Slack integration's receiving function does not need change to be compatible with Mattermost.
+2. The HTTP POST request body sent to a web service is formatted the same as Slack. This means your Slack integration's receiving function does not need change to be compatible with Mattermost.
```

Comment:
```suggestion
2. The HTTP POST request body sent to a web service is formatted the same as Slack's. This means your Slack integration's receiving function does not need change to be compatible with Mattermost.
```

Reply:
I'm reverting this as the original was accurate - it referred to the web service being the same format as that belonging to Slack.